client/src/site-search/search-results.scss

```diff
@@ -1,41 +1,115 @@
-.search-results {
-  a.title {
-    color: black;
-    font-weight: bold;
+@import "~@mdn/minimalist/sass/vars/color-palette";
+@import "~@mdn/minimalist/sass/vars/layout";
+@import "~@mdn/minimalist/sass/vars/borders";
+@import "~@mdn/minimalist/sass/vars/typography";
+
+@import "./pagination";
+
+.search-results-container {
+  .title {
+    background-color: inherit;
+    color: $neutral-100;
+    margin-bottom: $base-spacing / 4;
+    padding: 0;
+
+    a {
+      color: $neutral-100;
+    }
   }
-  span.highlight {
-    display: block;
+
+  .search-results-options {
+    background-color: $neutral-550;
+    padding: $base-spacing;
+
+    h2 {
+      font-size: $default-font-size;
+      margin-bottom: $base-spacing / 4;
+    }
+  }
+
+  .current-option {
     font-style: italic;
-    font-size: 0.9rem;
   }
-  span.summary {
+
+  .sort-options,
+  .search-language-options {
+    li {
+      display: inline;
+    }
+  }
+
+  .advanced-options {
+    margin-top: $base-spacing / 2;
+  }
+
+  .search-language-options {
+    font-style: italic;
+
+    a {
+      font-style: normal;
+    }
+  }
+
+  .search-results-list {
+    margin-bottom: $base-spacing;
+
+    mark {
+      background-color: $yellow-400;
+      font-style: italic;
+    }
+  }
+
+  .result-content-container {
+    color: $neutral-200;
+    margin-bottom: $base-spacing * 2;
+  }
+
+  .highlight,
+  .summary {
     display: block;
-    font-size: 0.9rem;
   }
-  span.nerd-data {
-    margin-left: 20px;
-    font-size: 80%;
+
+  .document-url {
+    margin-bottom: $base-spacing;
```

Comment:
I think it looks weird with this massive gap. If I squint and step back a bit it reads to me like:

1. Document A title and document URL
2. Document A extracts
3. Document B title and document URL
4. Document B extracts

When I'm expecting it to be:

1. Document A title and document URL and extracts
2. Document B title and document URL and extracts

Here's what it currently looks like:
<img width="796" alt="Screen Shot 2021-02-23 at 9 20 26 AM" src="https://user-images.githubusercontent.com/26739/108857411-0f3d3080-75b9-11eb-8d76-dfaac76ca7f6.png">

I used the devtools to change it to 6px and I think it looks much better:
<img width="861" alt="Screen Shot 2021-02-23 at 9 22 31 AM" src="https://user-images.githubusercontent.com/26739/108857457-1b28f280-75b9-11eb-8592-7572f53cbc7c.png">

I don't know if "6px" is the right answer nor am I a designer but it definitely feels better this way. The documents' title now feels more connected to the extracts/highlights rather than them floating all by themselves.